lang/cs/Org.Apache.REEF.Wake/Remote/Impl/DefaultRemoteManager.cs

```diff
@@ -79,7 +80,8 @@ public sealed class DefaultRemoteManager<T> : IRemoteManager<T>
             _server.Run();
 
             LocalEndpoint = _server.LocalEndpoint;
-            Identifier = new SocketRemoteIdentifier(LocalEndpoint);
+            IPEndPoint address = new IPEndPoint(localAddressProvider.LocalAddress, _server.LocalEndpoint.Port);
+            Identifier = new SocketRemoteIdentifier(address);
```

Comment:
Do we still need this change in .NET side?

Reply:
It is not needed. I changed this just to maintain parity with the java side. It is "nice to have" per recommendation from Sergiy.

---
In reply to: [200413195](https://github.com/sharathmalladi/reef/pull/6#discussion_r200413195) [](ancestors = 200413195)

Reply:
Will revert back to original. I had initially done this since this was "nice to have"  .. but I really don't have a way to validate this. So I will just ignore this for now.

---
In reply to: [200413195](https://github.com/sharathmalladi/reef/pull/6#discussion_r200413195) [](ancestors = 200413195)